Localization/ExplorerL10N.ini

```diff
@@ -1205,7 +1205,7 @@ Toolbar.InvertSelection = Inverti selezione
 Toolbar.FolderOptions = Opzioni cartella
 Toolbar.ShowHiddenFiles = Cartelle e file nascosti
 Toolbar.ShowSystemFiles = File di sistema
-Toolbar.ShowExtensions = Estensioni nomi file
+Toolbar.ShowExtensions = Mostra estensioni file
```

Comment:
Google Translate:
Estensioni nomi file - File name extensions
Mostra estensioni file - Show file extensions

Reply:
Should be 

"Visualizza estensioni file"

"Mostra" is not the right term in the context.